config/rabbitmq.yml

```diff
@@ -10,6 +10,12 @@ development:
     - processor: email_unpublishing_processor
       name: email_unpublishing
       routing_key: '*.unpublish.#'
+    - processor: subscriber_list_details_update_processor_major
+      name: subscriber_list_details_update_major
+      routing_key: "*.major.#"
+    - processor: subscriber_list_details_update_processor_minor
+      name: subscriber_list_details_update_minor
+      routing_key: "*.minor.#"
```

Comment:
The names of the processors, queues, rake tasks, and procfile workers should use a consistent ordering of parts.

So I think these processors should be called `subscriber_list_details_update_{major,minor}_processor` to fit in with the rest.